src/analyses/goto_check.h

```diff
@@ -29,4 +29,36 @@ void goto_check(
   const optionst &options,
   goto_modelt &goto_model);
 
+#define GOTO_CHECK_OPTIONS \
+  "(bounds-check)(pointer-check)(memory-leak-check)" \
+  "(div-by-zero-check)(signed-overflow-check)(unsigned-overflow-check)" \
+  "(pointer-overflow-check)(conversion-check)(undefined-shift-check)" \
+  "(float-overflow-check)(nan-check)"
+
+#define GOTO_CHECK_HELP \
+  " --bounds-check               enable array bounds checks\n" \
+  " --pointer-check              enable pointer checks\n" \
+  " --memory-leak-check          enable memory leak checks\n" \
+  " --div-by-zero-check          enable division by zero checks\n" \
+  " --signed-overflow-check      enable signed arithmetic over- and underflow checks\n" \
+  " --unsigned-overflow-check    enable arithmetic over- and underflow checks\n" \
+  " --pointer-overflow-check     enable pointer arithmetic over- and underflow checks\n" \
+  " --conversion-check           check loss of non-zero bits on type cast\n" \
```

Comment:
Is this description correct? If I have int i=-1; short j=(short)i; then I lose non-zero bits, but still nothing bad happens...